pkg/chain/config.go

```diff
@@ -0,0 +1,19 @@
+package eth
+
+// Config stores app-specific extensions configuration.
+type Config struct {
+	TBTC TBTC
+}
+
+// TBTC stores configuration of application extension responsible for
+// executing signer actions specific for TBTC application.
+type TBTC struct {
+	TBTCSystem string
+	BTCRefunds BTCRefunds
+}
+
+// BTCRefunds stores configuration related to recovering BTC from a closed keep.
+type BTCRefunds struct {
+	BeneficiaryAddress string
+	MaxFeePerVByte     int32
+}
```

Comment:
Why don't we leave this structure in `config/config.go`? Feels like a good place, especially that part of it is BTC specific and we [discussed](https://github.com/keep-network/keep-ecdsa/pull/693#discussion_r576261733) having a `bitcoin` namespace in the config.

Reply:
I tried that originally, but you end up creating circular dependencies!

`pkg/chain/config.go` needs to be able to reference a type, and if you reference `config.Extensions.TBTC`, then you have to import `config` from `chain`. That creates circles since `config` imports `chain` :(

Reply:
Oh my. I see it now. 

I'd suggest to introduce `pkg/chain/bitcoin/config.go` with
```
type Config struct {
	BeneficiaryAddress string
	MaxFeePerVByte     int32
}
```

This will be aligned with `config.Config` where we reference `ethereum.Config`, `libp2p.Config`, and `client.Config`. This will also be aligned with the previously referenced discussion about `bitcoin` namespace in the configuration file.

I'd consider leaving 
```
type Extensions struct {
	TBTC TBTC
}
``` 
inside `config/config.go`. `TBTCSystem` address is [now stored in ethereum config](https://github.com/keep-network/keep-ecdsa/blob/d6bedeaac806bb03a8af050063570a9631c5a444/cmd/connector_ethereum.go#L34) so you may save yourself headache when merging `master` changes into the integration branch later although for now, it will not look that elegant.

Reply:
i like it!